pages/ime.vue

```diff
@@ -56,10 +55,10 @@
       <PageBar>
         <template #left>
           <div class="my-10">
-            <h1 class="titulo txtBlack text-md-left">
+            <h1 class="titulo oIME text-md-left">
```

Comment:
As cores estavam meio desajustadas em relação ao Figma. Enviei alguns _commits_ corrigindo a coloração padrão do botão e do texto na cor preta. Para que fique uma cor padrão em todas as áreas do site, podemos usar a classe correspondente.

```suggestion
            <h1 class="titulo txtBlack text-md-left">
```